utoipa-gen/src/schema/into_params.rs

```diff
@@ -147,33 +214,65 @@ impl IntoParams {
             None => {
                 abort! {
                     ident,
-                    "struct with unnamed fields must have explisit name declarations.";
+                    "struct with unnamed fields must have explicit name declarations.";
                     help = "Try defining `#[into_params(names(...))]` over your type: {}", ident,
                 }
             }
         }
     }
 }
 
-struct Param<'a>(&'a Field, Option<&'a String>);
+#[cfg_attr(feature = "debug", derive(Debug))]
+pub struct FieldParamContainerAttributes<'a> {
```

Comment:
It is pretty good idea to have possibility to define the `style` in struct level for all parameter fields, but after all since it is parameter specific attribute. Is it still possible to override this struct level style with style in parameter level with `param(style = ...)`?

Reply:
https://github.com/juhaku/utoipa/pull/150/files#diff-bf76ef193c3d97fa4bbda36c667b8f030fe35fbbf0690dc8442bf231e7487043R284-R292 here the `style` container attribute will be overridden by the field attribute

Reply:
Line 284-292 in `into_params.rs` this file

Reply:
Good